maps-app-ios/UI/Feedback View/Feedback.storyboard

```diff
@@ -77,7 +77,7 @@
                                 <constraints>
                                     <constraint firstAttribute="height" constant="44" id="wvN-YM-V1y"/>
                                 </constraints>
-                                <textInputTraits key="textInputTraits" autocorrectionType="no" spellCheckingType="no" returnKeyType="search" enablesReturnKeyAutomatically="YES"/>
+                                <textInputTraits key="textInputTraits" autocorrectionType="no" spellCheckingType="no" returnKeyType="search" enablesReturnKeyAutomatically="YES" textContentType="location"/>
```

Comment:
Just wondering what this change does?

Reply:
Good question. That was generated automatically by XCode. Likely because i've upgraded to XC.10 .. ?